pandas/tests/io/formats/style/test_highlight.py

```diff
@@ -5,61 +5,68 @@
 
 pytest.importorskip("jinja2")
 
+from pandas.io.formats.style import Styler
 
-class TestStylerHighlight:
-    def setup_method(self, method):
-        np.random.seed(24)
-        self.s = DataFrame({"A": np.random.permutation(range(6))})
-        self.df = DataFrame({"A": [0, 1], "B": np.random.randn(2)})
 
-    def test_highlight_null(self):
-        df = DataFrame({"A": [0, np.nan]})
-        result = df.style.highlight_null()._compute().ctx
-        expected = {(1, 0): [("background-color", "red")]}
-        assert result == expected
+@pytest.fixture
+def df():
+    return DataFrame({"A": [0, np.nan, 10], "B": [1, None, 2]})
 
-    def test_highlight_null_subset(self):
-        # GH 31345
-        df = DataFrame({"A": [0, np.nan], "B": [0, np.nan]})
-        result = (
-            df.style.highlight_null(null_color="red", subset=["A"])
-            .highlight_null(null_color="green", subset=["B"])
-            ._compute()
-            .ctx
-        )
-        expected = {
-            (1, 0): [("background-color", "red")],
-            (1, 1): [("background-color", "green")],
-        }
-        assert result == expected
 
-    @pytest.mark.parametrize("f", ["highlight_min", "highlight_max"])
-    def test_highlight_minmax_basic(self, f):
-        expected = {
-            (0, 0): [("background-color", "red")],
-            (1, 0): [("background-color", "red")],
-        }
-        if f == "highlight_min":
-            df = -self.df
-        else:
-            df = self.df
-        result = getattr(df.style, f)(axis=1, color="red")._compute().ctx
-        assert result == expected
+@pytest.fixture
+def s(df):
```

Comment:
can you call this maybe styler, s is too short

Reply:
done